pkg/u8g2/src/csrc/u8x8_riotos.c

```diff
@@ -89,18 +102,18 @@ uint8_t u8x8_gpio_and_delay_riotos(u8x8_t *u8g2, uint8_t msg, uint8_t arg_int, v
             xtimer_nanosleep(arg_int * 100);
             break;
         case U8X8_MSG_GPIO_CS:
-            if (u8g2->pins_enabled & (1 << U8X8_PIN_CS)) {
-                gpio_write(u8g2->pins[U8X8_PIN_CS], arg_int);
+            if (NULL != u8x8_riot_ptr && GPIO_UNDEF != u8x8_riot_ptr->pin_cs) {
```

Comment:
Maybe you want to add some parenthesis here, it would improve reading and avoid trouble in the future.